package.json

```diff
@@ -30,12 +30,16 @@
     "cosmiconfig": "^5.0.2",
     "debug": "^3.1.0",
     "dedent": "^0.7.0",
-    "execa": "^0.9.0",
+    "del": "^3.0.0",
+    "execa": "^1.0.0",
     "find-parent-dir": "^0.3.0",
+    "g-status": "^2.0.2",
     "is-glob": "^4.0.0",
     "is-windows": "^1.0.2",
     "jest-validate": "^23.5.0",
-    "listr": "^0.14.1",
+    "listr": "^0.14.2",
+    "listr-update-renderer":
+      "https://github.com/okonet/listr-update-renderer/tarball/upgrade-log-update",
```

Comment:
Hi, would it be possible to get this released as a version on npm instead of depending on it through a loose git branch?

Reply:
Having issues to install anything that depends on lint-staged now. We are behind a corporate proxy here.

Reply:
Please use a npm published version!